sql/src/main/java/io/crate/operation/BaseImplementationSymbolVisitor.java

```diff
@@ -44,7 +44,7 @@ public BaseImplementationSymbolVisitor(Functions functions) {
     @Override
     public Input<?> visitFunction(Function function, C context) {
         FunctionIdent ident = function.info().ident();
-        final FunctionImplementation functionImplementation = functions.get(ident.name(), ident.argumentTypes());
+        final FunctionImplementation functionImplementation = functions.getBuiltin(ident.name(), ident.argumentTypes());
```

Comment:
I think all places that already have a Function symbol available should now use `getQualified` ?